rhel7/profiles/ospp.profile

```diff
@@ -138,9 +138,6 @@ selections:
     - accounts_passwords_pam_faillock_interval
     - var_accounts_passwords_pam_faillock_unlock_time=never
     - accounts_passwords_pam_faillock_unlock_time
-    - dconf_gnome_banner_enabled
-    - login_banner_text=usgcb_default
```

Comment:
Help me understand - we have `sshd_enable_warning_banner` enabled in the profile - do we need specific wording of the warning? Becase `login_banner_text` contains wording, that should be then applied through `banner_etc_issue` rule that is not enabled by the profile. Can you check the requirements?

Reply:
According to my greps, `sshd_enable_warning_banner` does not use `login_banner_text`, it just sets `sshd_config` to do `Banner /etc/issue`.

As far as the ccc/ospp profile and the wording is concerned, it was brought up before, the response from stakeholders being:
> The configuration annex has specific text. I am not using the text for DoD 
systems. It is text for everyone else. I really do not think SCAP should 
check the contents of the banner. It should check that there is a banner.

Reply:
Perfect, thanks!